support-workers/src/main/scala/com/gu/zuora/ZuoraService.scala

```diff
@@ -29,9 +32,11 @@ class ZuoraService(val config: ZuoraConfig, client: FutureHttpClient, baseUrl: O
   def getAccount(accountNumber: String): Future[GetAccountResponse] =
     get[GetAccountResponse](s"accounts/$accountNumber", authHeaders)
 
-  def getAccountFields(identityId: IdentityId): Future[List[DomainAccount]] = {
+  def getAccountFields(identityId: IdentityId, now: OffsetDateTime): Future[List[DomainAccount]] = {
+    val recentDays = 28
```

Comment:
since the step functions retry for 24 hours, I thought 28 days should be plenty

Reply:
comment please!